executors/src/eoa/error_classifier.rs

```diff
@@ -115,6 +118,10 @@ impl EoaErrorMapper {
                     EoaExecutionError::AccountError {
                         message: message.to_string(),
                     }
+                } else if msg_lower.contains("we are not able to process your request at this time") {
```

Comment:
I think we have a error code for this. NBD tho but the string might change, the code won't

Reply:
code (32603) is pretty generic, used for any internal server error